File: StepFunctionInvoke/csharp/src/AppStage.cs
using Amazon.CDK;
using Constructs;

namespace StepFunctionInvoke
{
    public class AppStage : Stage
    {
        internal AppStage(Construct scope, string id, IStageProps props = null) : base(scope, id, props)
        {
            new AppStack(this, "AppStack");
        }
    }
}

File: CFNStackSetsDeploy/javascript/lib/app-stage.js
const cdk = require('aws-cdk-lib');
const { AppStack } = require('./app-stack');

class AppStage extends cdk.Stage {
  constructor(scope, id, props) {
    super(scope, id, props);
    
    new AppStack(this, 'AppStack');
  }
}

module.exports = { AppStage };